.appveyor.yml

```diff
@@ -0,0 +1,13 @@
+version: '2.0.{build}'
+image: Visual Studio 2017
+before_build:
+  - nuget restore Parse.sln
+build_script:
+  - msbuild /verbosity:quiet "Parse.sln"
+test_script:
+  - .\Parse.Test\tools\OpenCover.4.6.519\tools\OpenCover.Console.exe -target:dotnet.exe -register:user -targetargs:""C:\projects\parse-sdk-dotnet\Parse.Test\bin\debug\Parse.Test.dll" /noshadow" -filter:"" -output:parse_sdk_dotnet_coverage.xml -oldstyle
+after_test:
+  - ps: |
+      $env:PATH = 'C:\msys64\usr\bin;' + $env:PATH
```

Comment:
Same here with c:

Reply:
That's how the original was made.

Reply:
Also, this is just configuration for AppVeyor. No one will be running it locally.